packages/Search/examples/bvh_driver/bvh_driver.cpp

```diff
@@ -11,13 +11,13 @@
 
 #include <DTK_LinearBVH.hpp>
 
+#include "../point_clouds/point_clouds.hpp"
```

Comment:
Maybe instead modify `CMakeLists`?

Reply:
I talked with @dalg24 about doing it this way

Reply:
On second thought, please follow @aprokop suggestion.